pages/home/home.ejs

```diff
@@ -2,108 +2,470 @@
+              <h3 class="h6 card-header font-weight-bold py-2">
+                Workspaces
+              </h3>
+              <div class="card-body py-2">
+                <p class="card-text">
+                  Work remotely in custom Docker containers via in-browser frontends
+                </p>
+              </div>
+              <ul class="list-group list-group-flush small font-italic">
+                <li class="list-group-item py-1">Visual Studio Code</li>
+                <li class="list-group-item py-1">Jupyter notebooks</li>
+                <li class="list-group-item py-1">RStudio</li>
+              </ul>
+              <div class="card-footer small font-weight-bold py-1 border-0">
+                <i class="fas fa-fw fa-hourglass-half" aria-hidden="true"></i>
+                Coming soon!
+              </div>
+            </div> <!-- card -->
+            <div class="card shadow">
+              <h3 class="h6 card-header font-weight-bold py-2">
+                And more &hellip;
+              </h3>
+              <div class="card-body py-2">
+                <p class="card-text">
+                  Check out the docs to see more elements or how to roll your own!
+                </p>
+                <a class="btn btn-outline-primary btn-sm btn-block" href="https://prairielearn.readthedocs.io/en/latest/elements">
+                  <i class="fas fa-fw fa-shapes" aria-hidden="true"></i>
+                  Use elements
+                </a>
+                <a class="btn btn-outline-primary btn-sm btn-block" href="https://prairielearn.readthedocs.io/en/latest/devElements">
+                  <i class="fas fa-fw fa-laptop-code" aria-hidden="true"></i>
+                  Write elements
                 </a>
-              </td>
-            </tr>
-            <% }); %>
-          </tbody>
-        </table>
+              </div>
+            </div> <!-- card -->
+          </div> <!-- card-columns -->
+        </div> <!-- container -->
+      </div> <!-- jumbotron -->
 
-        <% } %> <!-- course_instances.length -->
-      </div>
-    </div>
+      <div class="jumbotron bg-light mb-0">
+        <div class="container">
+          <h2>Ready to get started?</h2>
+          <p class="lead">
+            Start by requesting a course and creating some content. Join the Slack workspace for ad hoc
+            assistance or some random banter!
+          </p>
+          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="#">
+            <i class="fab fa-fw fa-wpforms" aria-hidden="true"></i>
+            Request your course
+          </a>
+          <a class="btn btn-outline-dark btn-lg mr-2 mb-2" href="#">
+            <i class="fas fa-fw fa-laptop-code" aria-hidden="true"></i>
+            Create your content
+          </a>
+          <a class="btn btn-outline-dark btn-lg mb-2" href="#">
+            <i class="fab fa-fw fa-slack" aria-hidden="true"></i>
+            Join our community
+          </a>
```

Comment:
More hrefs that need updating

Reply:
3c16d3b53ca7b493108799651df3dabeef87b274